src/tests/test_utils/common_test_utils/include/common_test_utils/file_utils.hpp

```diff
@@ -247,7 +247,7 @@ inline std::vector<std::string> splitStringByDelimiter(std::string paths, const
     return splitPath;
 }
 
-inline std::string getModelFromTestModelZoo(const std::string& relModelPath);
+std::string getModelFromTestModelZoo(const std::string& relModelPath);
```

Comment:
Some methods here should be replaced by `ov::util`

Reply:
My next PR changes namespaces